tests/query/parser/unit_tests/test_parse_mql_query_body.py

```diff
@@ -0,0 +1,1013 @@
+        get_dataset("generic_metrics"),
+        Query(
+            from_clause=Entity(
+                EntityKey.GENERIC_METRICS_DISTRIBUTIONS,
+                get_entity(EntityKey.GENERIC_METRICS_DISTRIBUTIONS).get_data_model(),
+            ),
+            selected_columns=[
+                SelectedExpression(
+                    "aggregate_value", f.sum(column("value"), alias="aggregate_value")
+                )
+            ],
+            array_join=None,
+            condition=and_cond(
+                f.equals(
+                    column("metric_id"), literal("d:transactions/duration@millisecond")
+                ),
+                in_cond(column("dist"), f.tuple(literal("dist1"), literal("dist2"))),
+            ),
+            groupby=None,
+            having=None,
+            order_by=None,
+            limitby=None,
+            limit=None,
+            offset=0,
+            totals=False,
+            granularity=None,
+        ),
+    ),
+    pytest.param(
+        'sum(`d:transactions/duration@millisecond`){dist:["dist1", "dist2"]}',
+        get_dataset("generic_metrics"),
+        Query(
+            from_clause=Entity(
+                EntityKey.GENERIC_METRICS_DISTRIBUTIONS,
+                get_entity(EntityKey.GENERIC_METRICS_DISTRIBUTIONS).get_data_model(),
+            ),
+            selected_columns=[
+                SelectedExpression(
+                    "aggregate_value", f.sum(column("value"), alias="aggregate_value")
+                )
+            ],
+            array_join=None,
+            condition=and_cond(
+                f.equals(
+                    column("metric_id"), literal("d:transactions/duration@millisecond")
+                ),
+                in_cond(column("dist"), f.tuple(literal("dist1"), literal("dist2"))),
+            ),
+            groupby=None,
+            having=None,
+            order_by=None,
+            limitby=None,
+            limit=None,
+            offset=0,
+            totals=False,
+            granularity=None,
+        ),
+    ),
+    pytest.param(
+        'sum(`d:transactions/duration@millisecond`){dist:["dist1", "dist2"]}',
+        get_dataset("generic_metrics"),
+        Query(
+            from_clause=Entity(
+                EntityKey.GENERIC_METRICS_DISTRIBUTIONS,
+                get_entity(EntityKey.GENERIC_METRICS_DISTRIBUTIONS).get_data_model(),
+            ),
+            selected_columns=[
+                SelectedExpression(
+                    "aggregate_value", f.sum(column("value"), alias="aggregate_value")
+                )
+            ],
+            array_join=None,
+            condition=and_cond(
+                f.equals(
+                    column("metric_id"), literal("d:transactions/duration@millisecond")
+                ),
+                in_cond(column("dist"), f.tuple(literal("dist1"), literal("dist2"))),
+            ),
+            groupby=None,
```

Comment:
Can you get rid of all these default parameters? That would shave a lot of lines off.

Reply:
Yeah, this would probably be best done in `query_repr`